src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: '',
```

Comment:
- Kindly, we don't need to initialize the `status` state as you already defined the `categories` state 👍